src/test/java/de/tum/in/www1/artemis/participation/ParticipationIntegrationTest.java

```diff
@@ -98,6 +97,7 @@
 import de.tum.in.www1.artemis.service.quiz.QuizScheduleService;
 import de.tum.in.www1.artemis.user.UserUtilService;
 import de.tum.in.www1.artemis.util.LocalRepository;
+import de.tum.in.www1.artemis.web.rest.dto.SelfLearningFeedbackRequestDTO;
 import de.tum.in.www1.artemis.web.rest.dto.QuizBatchJoinDTO;
```

Comment:
```suggestion
import de.tum.in.www1.artemis.web.rest.dto.QuizBatchJoinDTO;
import de.tum.in.www1.artemis.web.rest.dto.SelfLearningFeedbackRequestDTO;
```
The server style test failes because of this